src/css/story/cards.styl

```diff
@@ -6,12 +6,8 @@ card()
 // Default card bottom margin (desktop)
 $p = 160px
 +card()
-  border-style solid
```

Comment:
My perspective on design isn't worth that much, but I do think there's some value in having some visual effect that delineates the 'cards' from their surroundings, particularly on mobile, where the white card background blends in perfectly with the white background of the chart). Like a border (maybe even just on one side), or a slightly contrasting background color?

Reply:
Planning to give this a more thoughtful pass tonight and tomorrow. I'm mostly following the requests of the school's art director, who's the keeper of our graphic brand standards. I'm also not much of a designer – more of an implementer